drive/config-types.bal

```diff
@@ -88,113 +86,12 @@ public enum CredentialBearer {
     POST_BODY_BEARER
 }
 
-# Represents the configurations of the client used to call the introspection endpoint.
-#
-# + httpVersion - The HTTP version of the client
-# + customHeaders - The list of custom HTTP headers
-# + customPayload - The list of custom HTTP payload parameters
-# + auth - The client auth configurations
-# + secureSocket - SSL/TLS-related configurations
-public type ClientConfiguration record {|
-    Oauth2HttpVersion httpVersion = HTTP_1_1;
-    map<string> customHeaders?;
-    string customPayload?;
-    ClientAuth auth?;
-    SecureSocket secureSocket?;
-|};
-
 # Represents the HTTP versions.
 public enum Oauth2HttpVersion {
     HTTP_1_1,
     HTTP_2
 }
```

Comment:
This also can be removed.
SecureSocket record also can be removed.

Reply:
Updated